packages/fe-container/src/utils/plugins/PluginsLoaderFacade.ts

```diff
@@ -78,13 +79,14 @@ export const finish = (user: Partial<User>) => {
   start()
 }
 
-const pluginToQiankunMapper = (user: Partial<User>, basePath: string) => {
+const pluginToQiankunMapper = (user: Partial<User>, basePath: string, shared: Shared) => {
   return (plugin: InternalPlugin) => ({
     name: plugin.id,
     entry: plugin.pluginUrl || '',
     container: `#${MICROLC_QIANKUN_CONTAINER}`,
     activeRule: `${basePath}${plugin.pluginRoute || ''}`,
     props: {
+      ...(shared.props || {}),
```

Comment:
this or is not necessary, if shared.props is undefined, the spread operator already correctly manage it